internal/commands/inspect.go

```diff
@@ -0,0 +1,142 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/deislabs/cnab-go/action"
+	"github.com/docker/app/internal"
+	"github.com/docker/app/internal/cnab"
+	"github.com/docker/app/internal/inspect"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+type inspectOptions struct {
+	credentialOptions
+	pretty        bool
+	orchestrator  string
+	kubeNamespace string
+}
+
+func inspectCmd(dockerCli command.Cli) *cobra.Command {
+	var opts inspectOptions
+	cmd := &cobra.Command{
+		Use:   "inspect [OPTIONS] RUNNING_APP",
+		Short: "Shows installation and application metadata, parameters and the containers list of a running application",
+		Example: `$ docker app inspect my-running-app
+$ docker app inspect my-running-app:1.0.0`,
+		Args:   cli.RequiresMaxArgs(1),
```

Comment:
Is it really a MaxArgs? What if we only run `docker app inspect`?
If running app is optional, it should be visible in the usage: `"inspect [OPTIONS] [RUNNING_APP]"`